californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfig.java

```diff
@@ -192,6 +192,8 @@
 		public static final String TCP_CONNECTION_IDLE_TIMEOUT = "TCP_CONNECTION_IDLE_TIMEOUT";
 		public static final String TCP_CONNECT_TIMEOUT = "TCP_CONNECT_TIMEOUT";
 		public static final String TCP_WORKER_THREADS = "TCP_WORKER_THREADS";
+		public static final String BERT_OPTION = "BERT_OPTION";
```

Comment:
"option" is a used term in CoAP. Therefore I would prefer an other term/name. What's about "USE_BERT"?